.packit-copr-rpm.sh

```diff
@@ -6,8 +6,8 @@
 
 set -exo pipefail
 
-# Get Version from HEAD
-VERSION=$(awk -F'[""]' '/version=/ {print $(NF-1)}' setup.py)
```

Comment:
I think the former was better to be honest, much cleaner, one single process

Reply:
The old one matches to multiple values in pyproject.toml but if you think using awk is important I'll submit an alternative.

Reply:
Do you find this alternative acceptable @ericcurtin ?

Reply:
Never mind your version is fine